### ISSUE TITLE
Feature request:  table serialization 

### ISSUE THREAD

**Issue by NitroLine** [kuco23/pokerlib#9]
Hi, sometimes can be helpful to save current state of game into file or something.
For example, using pickle.

Now i solving this problem, by saving game in action and cards, then i replay game and replace cards when needed.

**Comment by kuco23** [comment]
Yeah, was thinking about this a while back, but doing it optimally would be a project. If you have any ideas, please share.
